cmd/gokeyless/initialize.go

```diff
@@ -71,19 +71,29 @@ func initAPICall(token *apiToken, csr string) ([]byte, error) {
 	}
 	defer resp.Body.Close()
 
-	apiResp := new(initAPIResponse)
+	apiResp := &initAPIResponse{}
+
+	if resp.StatusCode != http.StatusOK {
+		// Check if the returned body is JSON by attempting to unmarshal. If it
+		// fails, return the HTTP status code for debugging.
+		apiErr, err := json.Marshal(apiResp.Errors)
```

Comment:
This should `ioutil.ReadAll(resp.Body)` to a `body []byte`, then attempt to Unmarshal it to `apiResp`. If it fails, it should log the raw body and `resp.StatusCode`. Otherwise
```Go
if !apiResp.Success {		
	errs, _ := json.Marshal(apiResp.Errors)
	return nil, fmt.Errorf("api call failed: %s", errs)
}
```